### PR TITLE
chore: update to latest version

### PR DESCRIPTION
**Pull request by yizha1**
Update:
- Update to latest version `v1.2.0`

### CODE REVIEW

**Review by FeynmanZhou — DISMISSED** [review]
LGTM

File: layouts/partials/banner.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section class="news is-medium flex justify-center">
     <p class="is-size-3-mobile">
-      <a href="https://github.com/notaryproject/notation/releases/tag/v1.1.1" class="banner-link">Notation v1.1.1 is available</a>
+      <a href="https://github.com/notaryproject/notation/releases/tag/v1.1.1" class="banner-link">Notation v1.2.0 is available</a>
```
Comment (shizhMSFT) [inline layouts/partials/banner.html:3]: The link is incorrect

Comment (FeynmanZhou) [reply layouts/partials/banner.html:3]: Good catch

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**